### DISCUSSION TITLE
Stepper Driver alternative to TMC2225

### DISCUSSION THREAD

**Discussion by mycowx** [doudar/SmartSpin2k#270] [Q&A] [answered]
Gathering the pieces for my build but having trouble finding the TMC2225 domestically (in the US).  I'm guessing it's supply chain related, but until that gets resolved is there alterative drivers I can use as a drop in replacement?  I have some A4988 and DRV8825 laying around.  It looks like TCM2208/TCM2209 are still readily available.  Any suggestions?

**Comment by rjbutler** [comment, marked as answer]
I used the TMC2208 in mine

**Reply by bdattilo** [reply]
Does this require any re-configuring of the code? If so, is there a guide somewhere?

**Reply by doudar** [reply]
Should be plug and play
